release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -646,6 +646,11 @@ For clusters that run on {rh-openstack}, the network functions virtualization de
 
 These changes are reflected in simplified post-installation and network configuration documentation.
 
+[id="ocp-4-11-vsphere-unicast-default"]
+==== Installations on Red Hat OpenStack Platform, VMware vSphere, or oVirt now configure keepalived with unicast as the default
```

Comment:
@ahardin-rh do we want to use the attribute for OSP here?

Reply:
Yes, updating now